bitmap-team-TeamName/lib/bmp-file-helper.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const fs = require('fs');
+module.exports = exports = {};
+const filepath = `${__dirname}/../img/${process.argv[2]}`;
+const filepathNewfile = `${filepath.slice(0,filepath.length-4)}-new.bmp`;
+const testFilePath = `${filepath.slice(0,filepath.length-4)}-test.txt`;
+
+exports.getBitMap = function(cb) {
+  fs.readFile(filepath, function(err, data) {
+    if (err) throw err;
+    cb(data);
+  });
+};
+
+exports.newBitMap = function(data) {
```

Comment:
@npisciotti1 @radenska @sneakyneeks2 @darms i like the modularity and separate methods in this file but you may want to consider passing the file path into your `getBitMap` function so that this it is more dynamic and can be reused without a globally accessible variable within the file